WordPress/gutenberg
# Block Bindings: Register bindings sources in widgets screens

## What?
As reported [here](https://github.com/WordPress/gutenberg/pull/65713#issuecomment-2397180972), I'm registering the core sources and bootstrapping the server ones in the widgets screens.
 
## Why?
They should be available in that place as well.

## How?
Just using the core functions to do so.

## Testing Instructions
This one is a bit tricky to test because post meta fields are not available in the widgets context, custom sources bindings can't be created through the UI yet, and the widgets screens don't support the "Code Editor" I believe.

1. Install a classic theme. I've been using Twenty Seventeen.
2. Register a custom source in the server.
```
register_block_bindings_source(
	'test/custom',
	array(
		'label'              => 'Custom Source',
		'get_value_callback' => function () {},
	)
);
```
3. Go to a page or post -> Click on the three dots in the top-right corner -> Code editor -> and insert a block connected to this source:
```

<p>Fallback</p>

```
4. Go back to the Visual editor, click on the block options, and copy the block.

**Test Widgets screen**
5. Go to Appearance -> Widgets, and paste the copied block into any widget.
6. Check that the "Custom Source" label is shown. Previously, it showed the "Fallback" value.

**Test Customize Widgets screen**
5. Go to Appearance -> Customize -> Widgets, and paste the copied block into any widget.
6. Check that the "Custom Source" label is shown. Previously, it showed the "Fallback" value.
